lab2/lab2/index.py

```diff
@@ -8,8 +8,7 @@ def printL(req):
 
 @handler("af","POST")
 def answerpost(req):
-    data=post(req)
-    
-    return 200,{"Content-type":"text/html"},data #return_code,headers,code of the page(here it is html)
+    data=(str(post(req))).split('&')
```

Comment:
Again — operator spacing.
And also remove the unnecessary parentheses wrapping `str(post(req))`

Also, do you really want to treat your `data` as `bytes` and not as a `str`?